device/protocol_mqtt_v1.go

```diff
@@ -48,6 +48,18 @@ func (d *Device) initializeMQTTClient(brokerAddress string) error {
 	opts.SetTLSConfig(tlsConfig)
 
 	opts.SetOnConnectHandler(func(client mqtt.Client) {
+		if err := d.setupSubscriptions(); err != nil {
+			d.m.Disconnect(0)
+			fmt.Println(err)
```

Comment:
Make these a little more explicit to help understanding where the error is coming from: "Can't setup subscriptions: err"